### PR TITLE
entities and scripts both are updated

### PR DESCRIPTION
**Pull request by sohag16030**
```
cmsUser:
cmsUserId→ int (auto generated)
mobileNumber →string (unique,not null)
email→string(unique,nullable)
name→string(not null)
gender→ enum(male,female,others)
status→enum(active,inactive)
List<Address> addresses;
List<AcademicInfo> academicInfos;

Address:
addressId →int (auto generated)
addressType →enum(present,permanent)
Division division;
District district;
Upazila upazila;

AcademicInfo:
academicInfoId→ int (auto generated)
level→enum(psc,jsc,ssc)
grade→double(not null)
class → enum(1…10)
List<Subject> subjects;

Subject:
SubjectId→ int (auto generated)
name→String
nameLocal→String
List<AcademicInfo> academicInfos;

Division:
divisionId→ int (auto generated)
name→string (unique,not null)
nameLocal→string (unique,not null)
active→ Boolean(not null)
List<District> districts;

District:
districtId→ int (auto generated)
name→string (unique,not null)
nameLocal→string (unique,not null)
active→ Boolean(not null)
List<Upazila> upazillas;

Upazila:
upozillaId→ int (auto generated)
name→string (unique,not null)
nameLocal→string (unique,not null)
active→ Boolean(not null)
```

### CODE REVIEW

File: content-management-system/src/main/java/com/cms/example/cms/entities/Upazila.java [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.FetchType;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.JoinColumn;
-import jakarta.persistence.ManyToOne;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```
Comment (suvro47) [inline content-management-system/src/main/java/com/cms/example/cms/entities/Upazila.java]: wildcard import.

File: content-management-system/src/main/java/com/cms/example/cms/entities/Division.java [hunk 2]
```diff
@@ -47,6 +48,10 @@ public class Division {
     private Boolean active;
 
     @OneToMany(mappedBy = "division", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    private List<Address> addresses;
+
+    @OneToMany(mappedBy = "division", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JsonIgnore
```
Comment (suvro47) [inline content-management-system/src/main/java/com/cms/example/cms/entities/Division.java]: na jene use kortesi andaji. remove kor

File: content-management-system/src/main/java/com/cms/example/cms/entities/District.java [hunk 3]
```diff
@@ -54,8 +55,11 @@ public class District {
     private Division division;
 
     @OneToMany(mappedBy = "district", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
-    private List<Upazila> upazillas;
+    private List<Upazila> upazilas;
 
+    @OneToMany(mappedBy = "district", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JsonIgnore
```
Comment (suvro47) [inline content-management-system/src/main/java/com/cms/example/cms/entities/District.java]: bujhis eta?

Comment (sohag16030) [reply content-management-system/src/main/java/com/cms/example/cms/entities/District.java]: bujhi.. ekta entity thek ar ekta entity relationship kno bujhbo na..
just @ gula use kore table creation gula ektu valo kore notice korte hbe.. unidirectional relation.. bidirectional.. egla ektu valo kore bujhlei hbe..

File: content-management-system/src/main/java/com/cms/example/cms/entities/Division.java (hunk 2 above)
Comment (sohag16030) [reply content-management-system/src/main/java/com/cms/example/cms/entities/Division.java]: removed

File: content-management-system/src/main/java/com/cms/example/cms/entities/Upazila.java (hunk 1 above)
Comment (sohag16030) [reply content-management-system/src/main/java/com/cms/example/cms/entities/Upazila.java]: office er PC te eta configre kora nai.. fix kore disi..

**Pull request merged**